Romans/Roman Numerals Decoder/index.js

```diff
@@ -0,0 +1,19 @@
+function solution (roman) {
```

Comment:
I would suggest you to make the function name clear and meaningful based on what the function is doing